src/App/_moodsHelper.js

```diff
@@ -0,0 +1,30 @@
+const moods = [
```

Comment:
Might not include this file actually, I was planning to have the application be getting art by mood, and associating it with color but realized creating this data would be too time consuming for the time constraint of this project. If I have time, this will be an extension.